pkg/didcomm/protocol/route/support_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package route
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/dispatcher"
+	mockdispatcher "github.com/hyperledger/aries-framework-go/pkg/internal/mock/didcomm/dispatcher"
+	mockkms "github.com/hyperledger/aries-framework-go/pkg/internal/mock/kms"
+	mockstore "github.com/hyperledger/aries-framework-go/pkg/internal/mock/storage"
+	"github.com/hyperledger/aries-framework-go/pkg/kms"
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// mock route coordination provider
+type mockProvider struct {
+	openStoreErr error
```

Comment:
Error should probably be more generic, are you expecting only openStoreError to be used for testing

Reply:
Yes.